immer-es5.js

```diff
@@ -0,0 +1,244 @@
+import {isObject} from "util"
```

Comment:
I don't see where isObject is used. Is this a typo?

Reply:
yeah. auto import :(